content/en-us/reference/engine/classes/Accoutrement.yaml

```diff
@@ -5,23 +5,52 @@ memory_category: Instances
 summary: |
   An object that can attach to a player's character.
 description: |
-  An Accoutrement welds its child `Class.Part|part` called "Handle" to the
+  An Accoutrement welds its child `Class.BasePart|part` called "Handle" to the
```

Comment:
Is the name "Handle" of specific importance, or is it just being used to reference the BasePart throughout the rest of this text?

Reply:
> Is the name "Handle" of specific importance, or is it just being used to reference the BasePart throughout the rest of this text?

I'm not the author of this PR but Handle is the name observed by the engine. There are other classes that require a BasePart named Handle to get built-in welding behaviour; Tool is another example. It would be accurate to say that it's important to get the weld behaviour in question and to reference the BasePart.